### PR TITLE
Bugfix/px1007 fix strings

### PR DESCRIPTION
**Pull request by SENya1990**
Merge after ATR-470

### CODE REVIEW

File: src/Acuminator/Acuminator.Analyzers/Resources.resx [hunk 1]
```diff
@@ -492,4 +492,25 @@
...
     <value>Suppress the {0} diagnostic with Acuminator in a comment</value>
   </data>
+  <data name="PX1007Class" xml:space="preserve">
+    <value>class</value>
+  </data>
+  <data name="PX1007Delegate" xml:space="preserve">
+    <value>delegate</value>
+  </data>
+  <data name="PX1007Enum" xml:space="preserve">
+    <value>enum</value>
```
Comment (k-s-popova) [inline src/Acuminator/Acuminator.Analyzers/Resources.resx]: enumeration

Comment (SENya1990) [reply src/Acuminator/Acuminator.Analyzers/Resources.resx]: @k-s-popova are you sure? The enum is pretty wide-known term

Comment (k-s-popova) [reply src/Acuminator/Acuminator.Analyzers/Resources.resx]: Yes. We have enumeration in our API reference, and Microsoft uses enumeration in the description of the enum keyword: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/enum

Comment (SENya1990) [reply src/Acuminator/Acuminator.Analyzers/Resources.resx]: ok, I'll fix it

Comment (SENya1990) [reply src/Acuminator/Acuminator.Analyzers/Resources.resx]: done

**Pull request merged**